homeassistant/components/twinkly/light.py

```diff
@@ -332,3 +334,22 @@ async def async_update_current_movie(self) -> None:
         _LOGGER.debug("Current movie: %s", current_movie)
         if current_movie and "id" in current_movie:
             self._current_movie = current_movie
+
+    async def async_update_current_color(self) -> None:
+        """Update the current active color."""
+        current_color = await self._client.get_current_colour()
+        _LOGGER.debug("Current color: %s", current_color)
+        if int(current_color.get("code")) == 1000:
```

Comment:
Curious: would it make sense to make `1000` a constant (so other devs can easily understand what it is if necessary)?

Reply:
Yes. I have plans to refactor more of the code to do a better validation of the return values from the different calls.
Added it to const now, and will expand on that in a later PR.